_doc-gen-canonical/action.yml

```diff
@@ -61,34 +61,30 @@ runs:
       shell: bash
       run: |
 
-        has_canonical_tag() {
+        remove_canonical_tag() {
             local file=$1
-            grep -qP '<link\s+rel="canonical".*\/>' ${file}
+            grep -vP '<link\s+rel="canonical".*\/>' "$file" > "$file.tmp" && mv "$file.tmp" "$file"
         }
 
         add_canonical_tag() {
             local file=$1
             local filename=$(basename "$file")
             local relative_path=${file#*version/*/}
-            local baseurl="https://${{ inputs.cname }}/version/stable"
+            local baseurl="https://geometry.docs.pyansys.com/version/stable"
```

Comment:
```suggestion
            local baseurl="https://${{ inputs.cname }}/version/stable"
```

Reply:
Thanks! I was testing this locally and just copy-pasted this.